runtime/src/lib.rs

```diff
@@ -8,8 +8,11 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 pub mod apis;
 pub mod configs;
+mod genesis_config_presets;
 mod weights;
 
+extern crate alloc;
```

Comment:
This is because the runtime is a `no_std` crate that supports allocation (more specifically, we use `alloc` in the `genesis_config_presets.rs` file), and `alloc` is a "sysroot" crate which is distributed with Rust itself, so the 2018 Rust update that removed the need for `extern crate` syntax doesn't apply for it (more info [here](https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#an-exception))